third-party/realdds/doc/initialization.md

```diff
@@ -19,6 +19,19 @@ These initialization messages should only have effect on devices that are not al
 Once all streams have been received, the device is initialized and ready to use. See [Streaming](streaming.md).
 
 
+## "Atomic" Initialization
+
+
+Initialization may happen interspersed with regular notifications: for example, a server that's already serving data and notifications to an existing client may see another client and broadcast initialization messages.
+
+Only one set of initialization messages can be sent at a time. I.e., if a new client is detected while initialization messages for a previous client are still outgoing, a new set of messages must go out but only once the previous set is finished.
+
+While a set of initialization messages are outgoing, all other notifications must take a back seat and wait until the set is written out.
```

Comment:
I think that it is more important to send notifications to existing client than to new clients.

Reply:
As discussed, current implementation writes all notification messages as a block of messages, other messages will be written after the block

Reply:
Agreed, but I don't think it's any big or long ask to send the init messages as one set without other messages interspersed in-between. It's how the current adapter/notification-server behaves.